frameworks/agno/from-agno/agentuity.yaml

```
@@ -0,0 +1,81 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/agentuity/cli/refs/heads/main/agentuity.schema.json
+
+# ------------------------------------------------
+# This file is generated by Agentuity
+# You should check this file into version control
+# ------------------------------------------------
+
+# The version semver range required to run this project
+version: '>=0.0.141'
+# The ID of the project which is automatically generated
+project_id: proj_03a55070b58bde949917feacc48399b8
+# The name of the project which is editable
+name: from-agno
```

```suggestion
name: agno-from-agno
```

We use `[framework]-[projectName]`, e.g. `crewai-basic`. So I guess this would be `agno-from-agno`?